meteor/client/ui/App.tsx

```diff
@@ -168,6 +169,44 @@ export const App = translateWithTracker(() => {
 			}
 		}
 
+		private mountPWAFullScreenTrigger() {
+			document.addEventListener(
+				'mousedown',
+				(e) => {
+					document.documentElement
+						.requestFullscreen({
+							navigationUI: 'auto',
+						})
+						.then(() => {
+							document.addEventListener(
+								'fullscreenchange',
+								() => {
+									this.mountPWAFullScreenTrigger()
+								},
+								{
+									once: true,
+								}
+							)
+						})
+						.catch((e) => console.error('Could not get FullScreen when running as a PWA', e))
+
+					e.preventDefault()
+
+					// Use Keyboard API to lock the keyboard and disable all browser shortcuts
+					if ('keyboard' in navigator) {
+						//@ts-ignore
```

Comment:
Add a comment as described here right away, and we won't have to come back to this line when we down the line enable ban-ts-comment: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-ts-comment.md

Reply:
Good idea. I've added a description of what we're expecting, why, why it's fine, and where the definiton of the proposed Keyboard API is.

Reply:
I have a feeling our linter rules reject all `ts-ignore`, but do allow `ts-expect-error`, so that once it is no longer an error we know to remove the comment before it starts hiding a new real error